docs/README.md

```diff
@@ -1,12 +1,35 @@
 # Introduction
 
-An **in-toto attestation** is authenticated metadata about one or more
-software artifacts[^1]. The intended consumers are automated policy engines,
-such as [in-toto-verify] and [Binary Authorization].
+The **in-toto Attestation Framework**
 
-**TODO: Clean this up; add a docs index.**
+1.  defines a standard [format](../spec/v1.0-draft/) for attestations which
+    bind subjects, the artifacts being described, to arbitrary authenticated
+    metadata about the artifact
+2.  provides a set of [pre-defined statements](../spec/predicates/) for
```

Comment:
Statements are something specific in the format, so we probably want to use a different term here, like "pre-defined claims formats" or "pre-defined metadata formats".